.circleci/config.yml

```diff
@@ -0,0 +1,22 @@
+version: 2
+jobs:
+  build:
+    docker:
+      - image: 'circleci/node:latest'
+    steps:
+      - checkout
+      - run:
+          name: install
+          command: npm install
```

Comment:
Wanna cache this?

Reply:
let's not cache until it's needed (i.e too slow)